person.rb

```diff
@@ -23,7 +23,7 @@ def initialize(age, name: 'Unknown', parent_permission: true)
   end
 
   def can_use_services?
-    @parent_permission
+    @parent_permission || @age >= 18
   end
```

Comment:
[OPTIONAL] Good job implementing the last change I required. For maximum results, it could be better if you use the `of_age?` method that we have already defined. This way if we ever need to change the way the method is calculated we only have to do it in one place.

This follows principles as DRY, which you are already familiar with, or Single Responsibility Principle, which you will be studying more in-depth in the future.

Reply:
- thanks a lot, and good observation, was my mistake, however, it was solved and calls that function.
``
  def can_use_services?
    @parent_permission || of_age?
  end
``